2020.csv

```diff
@@ -19,3 +19,4 @@ EuroSciPy,2020-07-27,2020-07-31,"Bilbao, Spain",ESP,Bizkaia Aretoa,,,https://www
 GeoPython,2020-07-27,2020-07-31,"Bilbao, Spain",ESP,Bizkaia Aretoa,,,http://2020.geopython.net,,
 PyCon ES,2020-10-02,2020-10-04,"Granada, Spain",ESP,,,,https://2020.es.pycon.org/,,
 PyConDE & PyData Berlin 2020,2020-10-14,2020-10-16,"Berlin, Germany",DEU,,,,https://de.pycon.org,https://pretalx.com/pyconde-pydata-berlin-2019/me/submissions/,https://de.pycon.org/sponsors/
+PyConWeb 2020,2020-05-09,2020-05-10,"Munich, Germany",DEU,Microsoft Germany HQ,,2020-02-29,http://pyconweb.com/,https://www.papercall.io/pyconweb20,https://pyconweb.com/sponsoring
```

Comment:
Could you move this up, maybe between PyCon Israel ann Python Machine Learning Conference?

Reply:
Oh, sure, my bad! Thanks for noticing it!